webapp/javascript/components/FlameGraphRenderer.jsx

```diff
@@ -87,19 +90,62 @@ class FlameGraphRenderer extends React.Component {
         "Reset Flamegraph View"
       );
     }
-    this.props.actions.fetchJSON(this.props.renderURL);
+    this.fetchFlameBearerData(this.props.renderURL);
   }
 
-  componentDidUpdate(prevProps) {
-    if (prevProps.renderURL != this.props.renderURL) {
-      this.props.actions.fetchJSON(this.props.renderURL);
+  componentDidUpdate(prevProps, prevState) {
+    if (this.getParamsFromRenderURL(this.props.renderURL).name != this.getParamsFromRenderURL(prevProps.renderURL).name ||
```

Comment:
Explicitly defining params because in the future there will be three different set of params:
1. Global params the global timeline
2. Left `<FlameGraphRenderer />`
3. Right `<FlameGraphRenderer />

and we only want each component to update when its respective parameters change